lib/AST/ConformanceLookupTable.h

```diff
@@ -372,7 +372,8 @@ class ConformanceLookupTable {
 
   /// Resolve the given conformance entry to an actual protocol conformance.
   ProtocolConformance *getConformance(NominalTypeDecl *nominal,
-                                      ConformanceEntry *entry);
+                                      ConformanceEntry *entry,
+                                      LazyResolver *resolver);
```

Comment:
You can grab the lazy resolver from `nominal->getASTContext().getLazyResolver()`, rather than having to pass it along.

Reply:
Oh, I didn't realise that was okay, since the other places here seem to thread it through.